package.json

```diff
@@ -29,7 +29,8 @@
     "test": "node Makefile.js test",
     "test:cli": "mocha",
     "test:fuzz": "node Makefile.js fuzz",
-    "test:performance": "node Makefile.js perf"
+    "test:performance": "node Makefile.js perf",
+    "wdio": "wdio run ./wdio.conf.js"
```

Comment:
It doesn't match [our Package.json Conventions](https://github.com/eslint/eslint/blob/main/docs/src/contribute/package-json-conventions.md), can you modify it?

Reply:
ahh, good catch 👍  removed.